context.go

```diff
@@ -324,11 +324,12 @@ func checkRequiredFlags(flags []Flag, context *Context) requiredFlagsErr {
 			var flagPresent bool
 			var flagName string
 			for _, key := range strings.Split(f.GetName(), ",") {
-				if len(key) > 1 {
-					flagName = key
+				_key := strings.TrimSpace(key)
```

Comment:
Nit: Prefixing variables with `_` is not a common pattern in Go, and I don't think we have any variables named like these in this project.

If `_key` has no semantic difference from `key`, it's fine to just do:

```go
key = strings.TrimSpace(key)
```

Since Go isn't going to enforce immutability, and the first value is no longer needed.